core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -462,13 +462,14 @@ protected double parseSpeed( String str )
     }
 
     /**
-     * This method parses a string ala "00:00" (hours and minutes) or "0:00:00" (days, hours and
-     * minutes).
+     * This method parses a string ala 'hh:mm', format for hours and minutes 'mm', 'hh:mm' or 'hh:mm:ss'
+     * FIXME: Add support for ISO_8601 
```

Comment:
The problem is not that it can't be added. The problem are several wrongly mapped duration. I think that it was like mixing minute stuff with month, although it is stated in the wiki: *duration=PT71M - A duration of 71 minutes. This is not the same as P71M which denotes 71 months.*